### DISCUSSION TITLE
Open the HTML preview to full screen.

### DISCUSSION THREAD

**Discussion by anpuhach** [KDE/ghostwriter#823] [Ideas]
I often work with large MDs with complex syntax, so I need to preview them in order to read them faster. But the ghostwriter has a limit to its size.

It would be great if there was an option to switch the HTML preview to full screen.

**Comment by avin-madhu** [comment]
Theres always an option for a VS code extension called Markdown Preview Enhanced 
link : [MPE](https://marketplace.visualstudio.com/items?itemName=shd101wyy.markdown-preview-enhanced)
